diff -up BeanShell/build.xml.omv~ BeanShell/build.xml
--- BeanShell/build.xml.omv~	2019-05-18 22:37:49.337367315 +0200
+++ BeanShell/build.xml	2019-05-18 22:40:09.960577865 +0200
@@ -114,6 +114,7 @@
 			includes="**/*.java"
 			excludes="${excludes},**/bak/**"
 		>
+			<compilerarg line="--add-modules java.desktop"/>
 			<classpath>
 				<fileset refid="lib-fileset"/>
 			</classpath>
diff -up BeanShell/src/bsh/util/AWTConsole.java.omv~ BeanShell/src/bsh/util/AWTConsole.java
--- BeanShell/src/bsh/util/AWTConsole.java.omv~	2019-05-18 22:42:46.190075452 +0200
+++ BeanShell/src/bsh/util/AWTConsole.java	2019-05-18 22:45:16.142748226 +0200
@@ -220,8 +220,7 @@ public class AWTConsole extends TextArea
 		Great.  What a piece of crap.
 	*/
 	public void setCaretPosition( int pos ) {
-		((java.awt.peer.TextComponentPeer)getPeer()).setCaretPosition( 
-			pos + countNLs() );
+		super.setCaretPosition( pos + countNLs() );
 	}
 
 	/*
